frontend/apps/filemanager/filemanager.lua

```diff
@@ -301,7 +302,10 @@ function FileManager:init()
             },
         }
 
-        if not Device:isAndroid() and lfs.attributes(file, "mode") == "file" and util.isAllowedScript(file) then
+        -- checks if file is a python/shell script on non android devices
+        -- or if file is a shell script on android
+        if ( not Device:isAndroid() and lfs.attributes(file, "mode") == "file" and util.isAllowedScript(file) ) or
```

Comment:
Wouldn't it be more logical to add the Android exception to util.isAllowedScript() instead of wherever it may be used?

Reply:
We could add an argument to util.isAllowedScript(). If this argument is nil then the function acts as yet. IF it is there, only the listed script-types deliver true? 
The if on lines 307-308 would get smoother.

Reply:
Yep, +1 on adding an `isAndroid` flag to `util.isAllowedScript` to drop python.

(I mean, you can pass it a table of supported formats if you like, but that'd likely be a tad overengineered ^^).

Reply:
Passing it a table doesn't address my point, which is that you're defeating the purpose of util.isAllowedScript(). It's not the way lines 307 and 308 look that bothers me but what they do.